crux-mir/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# next
+
+## API changes
+
+* Much of the `crux-mir` library has been split off into a `crucible-mir`
+  library, which removes all of the `crux`-specific code. The following modules
+  were _not_ migrated to `crucible-mir`:
+
+  * `Mir.Generate`
+  * `Mir.Language`
+  * `Mir.Log`
+  * `Mir.Concurrency`
+  * `Mir.Overrides`
```

Comment:
At some point we may want to move certain overrides from `Mir.Overrides` into the crucible-mir library so we can call them from our patched version of the Rust standard library even when verifying with SAW.  But for now I think it's okay to leave it alone - we can move individual overrides later once we know which ones are actually necessary.